tests/test_bitset.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+from pyspextool.mc_bitset import bitset
```

Comment:
`mc_bitset.py` doesn't exist anymore so this script shouldn't work.

Before pushing a test, you should try it out by running `pytest` on your local machine. I'll post a tutorial in slack.